src/components/project_view/submission_detail/submission_detail_panel.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+  <div class="panel">
+    <div :class="[`${panel_correctness}-panel-header`,
+                  {'command-panel-header-open': d_is_open && is_command },
+                  {'panel-header-open': d_is_open && !is_command },
+                  {'panel-header-closed': !d_is_open }]"
+         @click="toggle_d_is_open">
+
+      <template v-if="is_command">
+          <div class="command-name">{{name}}</div>
+          <div class="command-correctness">
+            <correctness-icon :correctness_level="correctness_level">
+            </correctness-icon>
+          </div>
+      </template>
+
+      <template v-else>
+        <div class="name">{{name}}</div>
+        <div class="correctness">
+          <correctness-icon :correctness_level="correctness_level">
+          </correctness-icon>
+        </div>
+        <div class="points">
+          <span v-if="points_possible !== 0"
+                class="display-points">
+            {{points_awarded}}/{{points_possible}}
+          </span>
+        </div>
+      </template>
+    </div>
+
+    <div v-if="d_is_open"
+         :class="[`${panel_correctness}-panel-body`,
+                  {'multiple-command-panel-body': is_multi_command_case},
+                  {'command-panel-body': is_command}]">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
+
+import CorrectnessIcon, { CorrectnessLevel } from "@/components/project_view/submission_detail/correctness_icon.vue";
+
+export enum PanelCorrectnessLevel {
+    not_available = "not-available",
+    none_correct = 'none-correct',
+    some_correct = 'some-correct',
+    all_correct = 'all-correct',
+    info_only = 'info-only',
+    output_only = 'output-only',
+    dont_check = 'dont-check'
+}
+
+@Component({
+  components: {
+    CorrectnessIcon
+  }
+})
+export default class SubmissionDetailPanel extends Vue {
+  @Prop({required: true, type: String})
+  name!: string;
+
+  @Prop({required: true, type: String})
+  correctness_level!: CorrectnessLevel;
+
+  @Prop({default: 0, type: Number})
+  points_awarded!: number;
+
+  @Prop({default: 0, type: Number})
+  points_possible!: number;
+
+  @Prop({default: false, type: Boolean})
+  panel_is_active!: boolean;
+
+  @Prop({default: false, type: Boolean})
+  is_command!: boolean;
+
+  @Prop({default: false, type: Boolean})
+  is_multi_command_case!: boolean;
+
+  d_correctness_level: CorrectnessLevel = CorrectnessLevel.none_correct;
+  d_panel_is_active = false;
+  d_is_open = false;
+
+  @Watch('panel_is_active')
+  on_panel_is_active_change(new_value: boolean, old_value: boolean) {
```

Comment:
Does this value only ever potentially change when the user changes the adjust feedback setting? If that's the case, I'd argue that we don't want to watch panel_is_active so that whatever panels the user has open/closed stay that way when they change the feedback.

If all of that is the case and we make that change, then I think we should rename this variable to something like `open_initially`